### DISCUSSION TITLE
Can you make columns display vertically?

### DISCUSSION THREAD

**Discussion by bsamworthdev** [filamentphp/filament#9594] [Help]
My infolist currently has 4 textEntry items:

Section::make('Customer details')
                        ->columns(2)
                        ->schema([
                                TextEntry::make('Item 1'),
                                TextEntry::make('Item 2'),
                                TextEntry::make('Item 3'),
                                TextEntry::make('Item 4'),
])

...and it displays like this

+--------+---------+
|  Item 1   |  Item 2  |
+--------+---------+
|  Item 3   |  Item 4 |
+--------+---------+

Is there a way without rearranging the order of the items to make them stack vertically instead of horizontally?

i.e.
+--------+---------+
|  Item 1   |  Item 3  |
+--------+---------+
|  Item 2   |  Item 4 |
+--------+---------+

**Comment by leandrocfe** [comment]
I think you can't do it. Why not rearranging the order of the items?

**Reply by bsamworthdev** [reply]
Because then I can't make the number of columns dynamic by page size.

So if I use this code:

Section::make('Customer details')
->columns([
                    'sm' => 1,
                    'xl' => 2
                ])
->schema([
TextEntry::make('Item 1'),
TextEntry::make('Item 3'),
TextEntry::make('Item 2'),
TextEntry::make('Item 4'),
])

It will display ok on xl screens:

+--------+---------+
| Item 1    | Item 3     |
+--------+---------+
| Item 2    | Item 4     |
+--------+---------+

...but the order will be wrong on sm screens.

+--------+
| Item 1    | 
+--------+
| Item 3    |
+--------+
| Item 2    | 
+--------+
| Item 4    | 
+--------+
